### ISSUE TITLE
Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by CaptainXX** [jijiwuming/bilibilidownload#2]
```shell
PS D:\software\b站\bilibilidownload> node .\bilibilidownload.js
    \　　　/         /＼7　　　 ∠＿/
　   \　　/ 　   　 /　│　　 ／　／
　 ___\__/___　 　 │　Z ＿,＜　／　　 /`ヽ
  │　　　　　|　　│　　　　　ヽ　　 /　　〉
  | / 　   \ |  　 Y　　　　　`　 /　　/
  │　　　　　|　　ｲ●　､　●　　⊂⊃〈　　/
  │　\_/\_/  |　　()　 へ　　　　|　＼〈
  │　　　　　|　　　>ｰ ､_　 ィ　 │ ／／
   ￣￣￣￣￣ 　  / へ　　 /　ﾉ＜| ＼＼
                 ヽ_ﾉ　　(_／　 │／／
                 7　　　　　　　|／
                 ＞―r￣￣`ｰ―＿|
请输入b站视频地址:https://www.bilibili.com/bangumi/play/ep85829
D:\software\b站\bilibilidownload\bilibilidownload.js:64
                if(res.headers['content-encoding'].indexOf('gzip') != -1) {
                                                   ^

TypeError: Cannot read property 'indexOf' of undefined
    at ClientRequest.<anonymous> (D:\software\b站\bilibilidownload\bilibilidownload.js:64:38)
    at Object.onceWrapper (events.js:315:30)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:543:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:112:17)
    at Socket.socketOnData (_http_client.js:440:20)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
```

**Comment by jijiwuming** [comment]
bilibili's api has changed, i will update this repositorie this weekend @CaptainXX

**Comment by jijiwuming** [comment]
Fixed By f7ef308856a07fe159ed9d8e3943132fa6ead47d
